packages/wallet-sdk/src/store/store.ts

```diff
@@ -23,6 +23,15 @@ export type SubAccount = {
   address: Address;
   factory?: Address;
   factoryData?: Hex;
+  config?: {
+    capabilities: Record<string, unknown>;
+  };
```

Comment:
what capabilities are being stored here?

Reply:
any capabilties initialized from the config: i.e. sub accounts, spend limits, maybe paymaster

Reply:
none, forgot to remove it when i moved to a separate `SubAccountConfig` slice